src/main/java/net/minestom/server/command/builder/arguments/ArgumentType.java

```diff
@@ -146,13 +145,6 @@ public static ArgumentResourceLocation ResourceLocation(@NotNull String id) {
         return new ArgumentResourceLocation(id);
     }
 
-    /**
-     * @see ArgumentPotionEffect
-     */
-    public static ArgumentPotionEffect Potion(@NotNull String id) {
```

Comment:
Minestom is missing `minecraft:resource` and `minecraft:resource_or_tag` argument types. This is the replacement for the effect argument type.